e2e/cypress/integration/account_settings/general/nickname_spec.js

```diff
@@ -109,14 +114,84 @@ describe('Account Settings > Sidebar > General', () => {
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+            // # Click 'Add Members'
+            cy.get('#channelHeaderTitle').click();
+            cy.get('#channelAddMembers').click();
+
+            // * Verify that the username span contains the '@' symbol and the dark colour
+            cy.get('#multiSelectList > div > .more-modal__details > .more-modal__name > span').should('contain', '@').and('have.css', 'color', 'rgb(61, 60, 64)');
+
+            // # Close modal
+            cy.get('body').type('{esc}');
+
+            // # Go to manage members modal
+            cy.get('#channelMemberIcon').click();
+            cy.get('#member-list-popover').should('be.visible').within(() => {
+                cy.findByTestId('membersModal').click();
+            });
+
+            // * Verify that the username span contains the '@' symbol and the dark colour
+            cy.get('.more-modal__details > .more-modal__name').should('contain', '@').and('have.css', 'color', 'rgb(61, 60, 64)');
+
+            // # Close modal
+            cy.get('body').type('{esc}');
+
+            // # Click More... in the sidebar
+            cy.get('#alphaChannelList').should('be.visible').within(() => {
+                cy.findByText('More...').scrollIntoView().should('be.visible').click();
+            });
+
+            // # Go to direct messages modal
+            cy.get('.more-modal').should('be.visible').within(() => {
+                cy.findByText('Direct Messages').click();
+                cy.get('#selectItems').click().type('@');
+
+                // * Verify that the username span contains the '@' symbol and the dark colour
+                cy.get('.more-modal__details > .more-modal__name').should('contain', '@').and('have.css', 'color', 'rgb(61, 60, 64)');
+            });
+        });
+    });
+
+    it('MM-T2061 Nickname should reset on cancel of edit', () => {
+        cy.toAccountSettingsModal();
+
+        cy.get('#generalButton').should('be.visible').click();
+
+        // # Add the nickname to textfield contents
+        cy.get('#nicknameEdit').click();
+        cy.get('#nickname').clear().type('nickname_edit');
+
+        // # Cancel the edit of nickname
+        cy.get('#cancelSetting').click();
+
+        // # Click edit of nickname
+        cy.get('#nicknameEdit').click();
+
+        // * Check if element is present and contains old nickname
+        cy.get('#nickname').should('be.visible').should('contain', '');
+
+        cy.get('#accountSettingsHeader > .close').should('be.visible').click();
+    });
+
+    it('MM-T2062 Clear nickname and save', () => {
+        cy.toAccountSettingsModal();
+
         cy.get('#generalButton').should('be.visible').click();
 
         cy.get('#nicknameEdit').click();
         cy.get('#nickname').clear();
         cy.get('#saveSetting').click();
 
         cy.get('#nicknameDesc').should('be.visible').should('contain', "Click 'Edit' to add a nickname");
```

Comment:
Next step failed on me. Do you intend to click the "Edit" text?

Also, please add comments.

Reply:
I haven't added this test case, it's not part of the changes in the PR so I am not sure what they were trying to click but I can take a look. I will update the comments as well :)